recipes/configure-capture-agent-manager-supervisor.rb

```diff
@@ -0,0 +1,18 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-capture-agent-manager-supervisor
+
+include_recipe "mh-opsworks-recipes::update-package-repo"
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+install_package("supervisor")
+
+capture_agent_manager_info = node.fetch(:capture_agent_manager, {})
+app_name = capture_agent_manager_info.fetch(:capture_agent_manager_name, "capture_agent_manager")
+
+template %Q|/etc/supervisor/conf.d/#{app_name}.conf| do
+  source "capture-agent-manager-supervisor-conf.erb"
+  variables({
+    capture_agent_manager_name: app_name
+  })
+end
+
+execute %Q|service supervisor restart && supervisorctl start #{app_name}|
```

Comment:
i think i wasn't sure and run both just in case. i'm sure now since i've tested this a couple of times and it works how is supposed to. removed the chained `supervisor start <app>`.